app/views/mainstream_browse_pages/_redirects.html.erb

```diff
@@ -1,16 +1,29 @@
-<h3>Redirects</h3>
-
-<table class='table'>
-  <tr>
-    <th>Added</th>
-    <th>From</th>
-    <th>To</th>
-  </tr>
-  <% redirect_routes.order('created_at DESC').each do |redirect_route| %>
-    <tr>
-      <td><%= redirect_route.created_at.to_formatted_s(:long) %></td>
-      <td><%= link_to redirect_route.from_base_path, website_url(redirect_route.from_base_path), target: '_blank' %></td>
-      <td><%= link_to redirect_route.to_base_path, website_url(redirect_route.to_base_path), target: '_blank' %></td>
-    </tr>
-  <% end %>
-</table>
+<%= render "govuk_publishing_components/components/table", {
```

Comment:
Not entirely PR related, but:

What's the difference between using the Table Component vs. the Table Helper?
When/why would we want to use one over the other?

I can see we've used both in this PR, so I'm guessing there are certain situations that suit each best.

Reply:
So the table helper is what the table component is built with. We are using this as it allows us to use a specific style of the table (which is standard across the publishing apps). We will probably revise this in a future iteration and just stick to using GOVUK tables.